spec/controllers/alerts_controller_spec.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 require 'rails_helper'
+require "#{Rails.root}/app/presenters/special_users_presenter"
```

Comment:
These 'require' statements won't be necessary if the file name and class name follow the Rails convention.